### DISCUSSION TITLE
How to change the wallpaper from vanta waves to vanta net

### DISCUSSION THREAD

**Discussion by Mindtrip-Inno** [DustinBrett/daedalOS#255] [Q&A] [answered]
Nice twist. Job well done firstly,

I have tried to change wallpaper to vanta waves to vanta net. But copying  dist vanta.net.min.js ( in vanta github) to the vanta.waves.min.js gives black wallpaper. I believe the problem is some waves variables doesn't return valid with dist net.min.js code. Should i have to change everything in config related to waves or is there much easy way to do it ?

**Comment by DustinBrett** [comment]
Is there any errors in the console? With Vanta I had to upgrade it to work with the latest Three.js. Also the waves config settings might need changing.

**Comment by Mindtrip-Inno** [comment]
No error when running server. Just black screen. Min js is from vanta github.

![image](https://user-images.githubusercontent.com/71410539/211163973-a07c8a08-c266-42ef-a904-70f8ae76e0c1.png)

**Comment by DustinBrett** [comment, marked as answer]
Now that I think of it, ya you would need to change quite a bit to have it run in a worker which is what it's trying to do. I got it working by removing `VANTA` from the `WALLPAPER_WORKERS` constant so it never runs in a worker. Also in `vantaWaves/index.ts` you need to change references of `WAVES` to `NET`. Finally I had to replace my `three.min.js` which is very slimed down, with a full version of r148. It could probably look nicer if I also tweaked the config, but I left it untouched as it runs.

![image](https://user-images.githubusercontent.com/16656318/211168433-1a4ec503-710c-4843-91c3-bdd6e44d385d.png)

![image](https://user-images.githubusercontent.com/16656318/211168487-31776ecd-ed1e-45e7-8c4f-5e3d3608337f.png)

![image](https://user-images.githubusercontent.com/16656318/211168493-1816ea8a-37c2-42ea-834f-2219acffa25d.png)

![image](https://user-images.githubusercontent.com/16656318/211168496-42839563-b3f3-4fa5-a670-c62068f2bde7.png)
